pathauto.info.yml

```diff
@@ -4,10 +4,11 @@ core: 8.x
 type: module
 
 dependencies:
+- ctools
```

Comment:
Please add ctools also to .travis-before-script.sh, right next to token.module. Fixing all the tests is probably going to be quite a challenge.